cfme/tests/configure/test_access_control.py

```diff
@@ -604,7 +597,7 @@ def test_permissions_vm_provisioning():
 #    # Ensure VMs exist
 #    if not vms.get_number_of_vms():
 #        logger.debug("Setting up providers")
-#        setup_first_provider()
+#        infra_provider()
```

Comment:
Can fix but yes, it's commented out and it's been that way for a long time so I didn't dig into what's going on there.